org.spoofax.terms/src/org/spoofax/terms/io/TextTermWriter.java

```diff
@@ -0,0 +1,399 @@
+package org.spoofax.terms.io;
+
+import org.spoofax.interpreter.terms.*;
+import org.spoofax.terms.attachments.ITermAttachment;
+
+import java.io.IOException;
+
+/**
+ * Default term writer.
```

Comment:
What is a default term writer? Isn't this a term writer that writes to a textual ATerm format?